<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SSimu</class>
 <widget class="QDialog" name="SSimu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../Resources/pyleecan.qrc">:/images/images/MachineSetup/MachineType/machine_SynRM.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="g_OP">
         <property name="title">
          <string>Operating Point</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="in_N0">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Stator external radius</string>
            </property>
            <property name="text">
             <string>N0:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="FloatEdit" name="lf_N0">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Stator external radius</string>
            </property>
            <property name="text">
             <string>3000</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="unit_N0">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>[rpm]</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="in_I1">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Stator external radius</string>
            </property>
            <property name="text">
             <string>Id:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="FloatEdit" name="lf_I1">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Stator external radius</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="unit_I1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>[Arms]</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="in_I2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Stator external radius</string>
            </property>
            <property name="text">
             <string>Iq:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="FloatEdit" name="lf_I2">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Stator external radius</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="unit_I2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>[Arms]</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="in_I3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Stator external radius</string>
            </property>
            <property name="text">
             <string>If:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="FloatEdit" name="lf_I3">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Stator external radius</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="unit_I3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>[Arms]</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="in_Tmag">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Stator external radius</string>
            </property>
            <property name="text">
             <string>Tmag</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="FloatEdit" name="lf_Tmag">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Stator external radius</string>
            </property>
            <property name="text">
             <string>20</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="unit_Tmag">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>[degC]</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="g_mag">
         <property name="title">
          <string>Magnetic Model</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="in_Na_tot">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Stator external radius</string>
              </property>
              <property name="text">
               <string>Angular discretization:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="si_Na_tot"/>
            </item>
            <item row="0" column="2">
             <widget class="QCheckBox" name="is_per_a">
              <property name="text">
               <string>Angular periodicity</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="in_Nt_tot">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Stator external radius</string>
              </property>
              <property name="text">
               <string>Time discretization:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="si_Nt_tot"/>
            </item>
            <item row="1" column="2">
             <widget class="QCheckBox" name="is_per_t">
              <property name="text">
               <string>Time periodicity</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="in_Kmesh">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Stator external radius</string>
                </property>
                <property name="text">
                 <string>Mesh Fineness</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="FloatEdit" name="lf_Kmesh">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Stator external radius</string>
                </property>
                <property name="text">
                 <string>1</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="in_nb_worker">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Stator external radius</string>
                </property>
                <property name="text">
                 <string>Nb worker</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="si_nb_worker"/>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="g_out">
         <property name="title">
          <string>Results</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="WPathSelector" name="w_path_result" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="is_losses">
            <property name="text">
             <string>Compute Losses</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="is_mesh_sol">
            <property name="text">
             <string>Save Mesh Solution</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="b_previous">
       <property name="text">
        <string>Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b_next">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FloatEdit</class>
   <extends>QLineEdit</extends>
   <header>.....GUI/Tools/FloatEdit.h</header>
  </customwidget>
  <customwidget>
   <class>WPathSelector</class>
   <extends>QWidget</extends>
   <header>.....GUI.Tools.WPathSelector.WPathSelector</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Resources/pyleecan.qrc"/>
  <include location="../../../Resources/pyleecan.qrc"/>
 </resources>
 <connections/>
</ui>
